packages/mdc-list/index.js

```diff
@@ -78,6 +79,21 @@ class MDCList extends MDCComponent {
       .forEach((ele) => ele.setAttribute('tabindex', -1));
   }
 
+  initializeListType() {
+    // Automatically set single selection if selected/activated classes are present.
+    const preselectedElement =
+      this.root_.querySelector(`.${cssClasses.LIST_ITEM_ACTIVATED_CLASS}, .${cssClasses.LIST_ITEM_SELECTED_CLASS}`);
+
+    if (preselectedElement) {
+      if (preselectedElement.classList.contains(cssClasses.LIST_ITEM_ACTIVATED_CLASS)) {
+        this.foundation_.setUseActivatedClass(true);
+      }
+
+      this.singleSelection = true;
```

Comment:
is it safe to assume user wants to have single selection mode when list has selected item? 

Multi-selectable list can initially have one item selected.

Reply:
I think it is safe to assume that the user has a single selection list when using the selected or activated classes. There isn't a multi-selectable list, but this can be changed if we implement one.